breakingChangePost.md

```diff
@@ -0,0 +1,58 @@
+An upcoming release of the **AWS SDK for Kotlin** will introduce an optional region provider parameter that determines the AWS region when configuring a client
+
+# Release date
+
+This change will be included in the upcoming **v1.5.0** release,
+
+# What's changing
+Adding an optional `regionProvider` parameter to provide more flexibility in how regions are specified.
```

Comment:
This is not a complete sentence. "This release includes a new optional parameter..."